infra/examples/gcp-google-workspace/main.tf

```
@@ -10,12 +10,39 @@ terraform {
   }
 }
 
+locals {
+  base_config_path = "${var.psoxy_base_dir}/configs/"
+  bulk_sources = {
+    "hris" = {
+      source_kind = "hris"
+      rules = {
+        columnsToRedact = []
+        columnsToPseudonymize = [
+          "email",
+          "employee_id"
+        ]
+      }
+    },
+    "qualtrics" = {
+      source_kind = "qualtrics"
+      rules = {
+        columnsToRedact = []
+        columnsToPseudonymize = [
+          "email",
```

This qualtrics file probably doesn't matter that much, but ideally should be :point_down: to follow our conventions
```suggestion
          "employee_email",
```